bin/README.mustache

```diff
@@ -49,6 +49,27 @@ which is the authentication library used by all Google Cloud Node.js clients.
 - [gstore-node][gstore-node] - Google Datastore Entities Modeling library.
 - [gstore-api][gstore-api] - REST API builder for Google Datastore Entities.
 
+## Supported Node.js Versions
+
+Our client libraries follow [Node.js' release schedule](https://nodejs.org/en/about/releases/).
+Libraries are compatible with all current _active_ and _maintenance_ versions of
+Node.js.
+
+Client libraries targetting end-of-life versions of Node.js are available, and
```

Comment:
Maybe “may be available” instead of “are available” but this is a nit and it works either way.